sdcm/utils/latency.py

```diff
@@ -0,0 +1,109 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2020 ScyllaDB
+
+import time
+from sdcm.db_stats import PrometheusDBStats
+
+
+def avg(values):
+    if values:
+        return float(format(sum([float(val) for val in values])/len(values), '.1f'))
+    else:
+        return None
+
+
+def collect_latency(monitor_node, start, end, load_type):
+    res = dict()
+    prometheus = PrometheusDBStats(host=monitor_node.external_address)
+    time.sleep(5)
```

Comment:
why there a sleep here ? What are we waiting for ?

Reply:
to give the time to the Prometheus connection to be open and functional

Reply:
Should be part of PrometheusDBStats object, and there isn't any persistent connection that I'm aware of it in there

Reply:
so you mean here to add this delay while creating the `PrometheusDBStats` object?